libs/components/indicators/src/lib/modules/status-indicator/status-indicator.component.ts

```diff
@@ -31,12 +31,10 @@ export class SkyStatusIndicatorComponent {
    */
   @Input()
   public set indicatorType(value: SkyIndicatorIconType) {
-    this._indicatorType = value;
-    this.updateIcon();
-  }
+    this.indicatorTypeOrDefault =
```

Comment:
Same feedback here as on the label component.

Reply:
Dome